src/components/ui/form/button/style.scss

```diff
@@ -0,0 +1,16 @@
+@use "@styles/_base.scss";
+
+.btn {
+    align-items: center;
+    background-color: base.$bg-white;
+    border: 1px solid base.$bg-gray-400;
+    border-radius: 0.5rem;
+    color: base.$bg-slate-700;
+    display: inline-flex;
```

Comment:
whats the purpose of `inline` here?

Reply:
Here, I want the button element to flow `inline` with the text content. Thank you.

Reply:
can't we make this with `flex` only?

Reply:
Changed. Thank you.